tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "allowJs": false,
     "jsx": "react",
     "moduleResolution": "node",
-    "rootDirs": ["src", "stories"],
+    "rootDirs": ["src"],
```

Comment:
@dfcowell do we need stories when using the lib on the wallet? Isn't stories just for dev and perhaps deploy it to a gh-page so to see the components? Cheers

Reply:
@dfcowell strange. Dev is working fine for me without it and the integration with angular too. If I add it back it will add the stories files to the build/lib folder which I don't think it makes sense.